inference-engine/thirdparty/clDNN/tests/test_utils/test_utils.cpp

```diff
@@ -287,12 +287,20 @@ std::vector<std::shared_ptr<test_params>> generic_test::generate_generic_test_pa
 
 cldnn::engine_configuration get_test_engine_config() {
     const bool enable_profiling = false;
+#ifdef ENABLE_ONEDNN_FOR_GPU
+    const cldnn::queue_types queue_type = cldnn::queue_types::in_order;
```

Comment:
What about leaving some comment through cout so that we can be aware of this limitation in the future as well?

Reply:
Actually, I think we can keep default engine for all existing tests, and use custom one for onednn-based primitives.